sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1257,6 +1257,42 @@ public void testWriteValidatesDatasetStreaming() throws Exception {
   }
 
   @Test
+  public void testStreamingWriteFnCreateNever() throws Exception {
+    BigQueryIO.StreamingWriteFn fn = new BigQueryIO.StreamingWriteFn(
+        null, CreateDisposition.CREATE_NEVER, new FakeBigQueryServices());
+    assertEquals(BigQueryIO.parseTableSpec("dataset.table"),
+        fn.getOrCreateTable(null, "dataset.table"));
+  }
+
+  @Test
+  public void testCreateNeverWithStreaming() throws Exception {
+    BigQueryOptions options = TestPipeline.testingPipelineOptions().as(BigQueryOptions.class);
+    options.setProject("project");
+    options.setStreaming(true);
+    Pipeline p = TestPipeline.create(options);
+
+    TableReference tableRef = new TableReference();
+    tableRef.setDatasetId("dataset");
+    tableRef.setTableId("sometable");
+
+    PCollection<TableRow> tableRows =
```

Comment:
This creates a bounded collection, which fails the UNBOUNDED conditionals.  This raises a few possibilities:
- Should CreateUnbounded be a shorthand we have?
- Should these conditionals check options.isStreaming() (guessing not, because that's less clean with respect to the model)